packages/application-shell/src/utils/http-client.ts

```diff
@@ -150,6 +156,12 @@ const getAppliedForwardToHeaders = (
   if (!forwardToConfig.uri) {
     throw new Error(`Missing required "uri" option.`);
   }
+
+  const exchangeTokenClaims: TForwardToExchangeTokenClaim[] = [];
+  if (forwardToConfig.includeUserPermissions) {
+    exchangeTokenClaims.push('permissions');
```

Comment:
Good work @CarlosCortizasCT 🦾. 
Just curious, So after checking that the forwardToConfig has the `includeUserPermissions` property, we then add `permissions` into `exchangeTokenClaims`. I assume this is later used to build the exchangeTokenClaims right?

Reply:
Yes.
So `exchangeTokenClaims` will include any custom claim we should include in the new `X-Forward-To-Claims` request header.
That header is added to the headers list that will be included in the request [in this same file](https://github.com/commercetools/merchant-center-application-kit/blob/7e19d7ba31c513e37eb3dc56bccf041c68104c99/packages/application-shell/src/utils/http-client.ts#L179).

Reply:
Thanks for clarifying 🙏